media/js/src/AnnotationScroller.jsx

```diff
@@ -41,12 +41,22 @@ export default class AnnotationScroller extends React.Component {
     render() {
         const selectedAnnotation =
               this.props.annotations[this.state.currentAnnotation];
+
+        let assetLink = '#';
+        if (window.MediaThread) {
+            const courseId = window.MediaThread.current_course;
+            // TODO: link to annotation, not just asset.
+            assetLink =
+                `/course/${courseId}/react/asset/` +
+                selectedAnnotation.asset_id;
```

Comment:
Does this need a trailing slash?

Reply:
no - it doesn't have on in the other place I'm defining this, in GridAsset.jsx.